core/modules/block/block.admin.inc

```diff
@@ -116,6 +116,7 @@ function block_admin_configure($form, &$form_state, $delta = NULL) {
 
   // Call our own block configuration form (also used by Layout module).
   $form += block_custom_block_form($custom_block, TRUE);
+  $form['#attributes']['class'][] = 'sticky-enabled';
```

Comment:
There are other things that can be sticky, such as table headers and the admin bar menu. Should we use a more specific class name?:

```suggestion
  $form['#attributes']['class'][] = 'sticky-submit-enabled';
```